src/Stats.AzureCdnLogs.Common/CdnLogEntryParser.cs

```
@@ -33,55 +33,65 @@ public static CdnLogEntry ParseLogEntryFromLine(string line)
 
-            // cs-method
-            TrySetStringProperty(value => entry.HttpMethod = value, columns[8]);
+                // cs-uri-stem
+                TrySetStringProperty(value => entry.RequestUrl = value, columns[9]);
 
-            // cs-uri-stem
-            TrySetStringProperty(value => entry.RequestUrl = value, columns[9]);
+                // skip column 10, it just contains the '-' character
 
-            // skip column 10, it just contains the '-' character
+                // rs-duration
+                TrySetIntProperty(value => entry.RemoteServerTimeTaken = value, columns[11]);
 
-            // rs-duration
-            TrySetIntProperty(value => entry.RemoteServerTimeTaken = value, columns[11]);
+                // rs-bytes
+                TrySetLongProperty(value => entry.RemoteServerBytesSent = value, columns[12]);
 
-            // rs-bytes
-            TrySetLongProperty(value => entry.RemoteServerBytesSent = value, columns[12]);
+                // c-referrer
+                TrySetStringProperty(value => entry.Referrer = value, columns[13]);
 
-            // c-referrer
-            TrySetStringProperty(value => entry.Referrer = value, columns[13]);
+                // c-user-agent
+                TrySetStringProperty(value => entry.UserAgent = value, columns[14]);
 
-            // c-user-agent
-            TrySetStringProperty(value => entry.UserAgent = value, columns[14]);
+                // customer-id
+                TrySetStringProperty(value => entry.CustomerId = value, columns[15]);
 
-            // customer-id
-            TrySetStringProperty(value => entry.CustomerId = value, columns[15]);
+                // x-ec_custom-1
+                TrySetStringProperty(value => entry.CustomField = value, columns[16]);
+            }
+            catch (FormatException e)
+            {
+                // skip this line but log the error
+                onErrorAction?.Invoke(e);
```

I prefer people passing in a null pointer instead of an empty lambda :)
I think the real ask here is whether we should just let the method throw the FormatException when the onErrorAction = null? 
Will adjust to let it throw when no error handler provided, unless you have a strong preference one way or the other.

See https://github.com/NuGet/NuGet.Jobs/commit/3190aa4388151767d9c7989c1f1368a3d6f843f3